### PR TITLE
Add soft failure to restart network after set hostname

### PR DESCRIPTION
**Pull request by jknphy**
br0 is stuck in setup-in-progress. Reload and Restart fixes the issue. Screenshots included to see that finishes in "up" status.

- Related ticket: https://progress.opensuse.org/issues/28603
- Verification run: http://dhc227/tests/585#

### CODE REVIEW

**Comment by okurz** [comment]
It's a nice idea but it looks like you just want to make the test green with any mean possible. The issue you are facing looks exactly like the product bug we track already in the xen and allpatterns scenario

**Comment by nicksinger** [comment]
I also think that this is a suitable workaround but not a real fix for the problem. Could you change your code to detect these fails and apply the workaround?

**Comment by okurz** [comment]
https://bugzilla.suse.com/show_bug.cgi?id=1061051 should be the bug

**Comment by jknphy** [comment]
The reload-or-restart command is replaced by something similar that works. Should I add a soft failure referencing the bug? The idea to make it green is to move forward for next bugs and due to this one is new for SLE15, it just started in one failing needle and also contains milestones, it was not a bad idea to by-pass as much as possible, but I forgot to create a bug for that. Thanks for reference the existing one.

**Comment by nicksinger** [comment]
+1 for a soft-fail. IMHO that's their sole purpose: move forward while still being aware of existing bugs.

File: tests/console/hostname.pm [hunk 1]
```diff
@@ -28,6 +28,17 @@ sub run {
     script_run "systemctl --no-pager status network.service";
     save_screenshot;
     assert_script_run "if systemctl -q is-active network.service; then systemctl reload-or-restart network.service; fi";
+    if (script_run("ip addr show br0 | grep DOWN") == 0) {
+        record_soft_failure('bsc#1061051');
+        assert_script_run "systemctl reload network";
+        script_run "systemctl --no-pager status network.service";
```
Comment (okurz) [inline tests/console/hostname.pm]: We have a `systemctl` wrapper command in `lib/utils`. Can you use that one please?

**Comment by jknphy** [comment]
Using wrapper now.
VR: http://dhcp227tests/625#step/hostname/34

**Pull request merged**